var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -271,6 +271,7 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
     patch('glibc-2.31-libsanitizer-2.patch', when='@8.1.0:8.3.0,9.0.0:9.2.0')
     patch('glibc-2.31-libsanitizer-2-gcc-6.patch', when='@5.3.0:5.5.0,6.1.0:6.5.0')
     patch('glibc-2.31-libsanitizer-2-gcc-7.patch', when='@7.1.0:7.5.0')
+    patch('glibc-2.31-libsanitizer-3.patch', when='@:11.1.0')
```

Comment:
Would it be possible to fetch this patch from some url? Also, are all these patches modifying the same behavior across different GCC versions?

Reply:
Hi @alalazo, in this address there is a link for the patch (https://gcc.gnu.org/git/gitweb.cgi?p=gcc.git;h=2b40941d23b1570cdd90083b58fa0f66aa58c86e), but I do not know, if it is permanent. In regard to the second question, I can not assert for all the versions available in Spack, I have checked it with three different versions < 11.1.0. Do we need some more test cases?

Reply:
Let's use this url or something similar:
- https://gcc.gnu.org/git/?p=gcc.git;a=commitdiff;h=2b40941d23b1570cdd90083b58fa0f66aa58c86e;hp=e9aeff6adbd8655ef3165cada15fda5fb6267551

and delete the patch from Spack repo. Even if it is not permanent, we'll cache it in our source mirror when it gets updated. 

>  In regard to the second question, I can not assert for all the versions available in Spack, I have checked it with three different versions < 11.1.0. Do we need some more test cases?

The question is just to understand if all the patch directives called `glibc-2.31-libsanitizer-*` do the same thing to different versions of GCC (I admittedly didn't take the time to have a look at that myself, sorry). If not, please put a brief comment above this patch to explain what it changes.

Reply:
Hi,
I already did it. One question, should I create a new  merge request or there is a way to push the new commit to this MR?

Reply:
@acastanedam The PR is fine like it is, thanks. I'll be waiting for tests to finish and merge it.

Reply:
Alright, thanks to you